### PR TITLE
add cortex docs

### PR DESCRIPTION
**Pull request by cristinabuenahora**
**Requirements**

- [x] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [x] I have updated my user documentation
- [x] I have updated my developer documentation (my integration README)
- [x] I have submitted the [Become a Partner](https://launchdarkly.com/partners/) form

**Intended availability**

Partner integration available to anyone who is both a Cortex & LaunchDarkly customer

**Related issues**

N/A

**Describe the solution you've provided**

Integration docs: https://docs.cortex.io/docs/reference/integrations/launchdarkly

**Describe alternatives you've considered**

N/A

**Provide a request preview**

```
> @launchdarkly/integration-framework@1.0.0 preview
> node preview.js cortex

The "cortex" integration does not make any outbound requests from LaunchDarkly.
```

**Additional context**

N/A

### CODE REVIEW

File: integrations/cortex/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Cortex",
+  "version": "1.0.0",
+  "overview": "View & enforce usage of LaunchDarkly feature flags directly in the catalog and Scorecards in Cortex.",
+  "description": "With the integration, you can view and monitor feature flags in the catalog. You can also write Scorecard rules to enforce feature flag usage and drive operational maturity.",
+  "author": "Cortex",
+  "supportEmail": "support@cortex.io",
+  "links": {
+    "site": "https://www.cortex.io/home",
+    "launchdarklyDocs": "https://docs.cortex.io/docs/reference/integrations/launchdarkly",
```
Comment (InTheCloudDan) [inline integrations/cortex/manifest.json:10]:
```suggestion
    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/cortex",
```

**Review by InTheCloudDan — CHANGES_REQUESTED** [review]
@cristinabuenahora just a few minor changes and we can get this merged. The docs are already live.

**Review by scribblingfox — APPROVED** [review]

File: integrations/cortex/manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "description": "With the integration, you can view and monitor feature flags in the Cortex catalog. You can also write Scorecard rules to enforce in Cortex feature flag usage and drive operational maturity.",
+  "author": "Cortex",
+  "supportEmail": "support@cortex.io",
+  "links": {
+    "site": "https://www.cortex.io/home",
+    "launchdarklyDocs": "https://docs.launchdarkly.com/integrations/cortex",
+    "supportWebsite": "https://support.getcortexapp.com/",
+    "privacyPolicy": "https://www.cortex.io/legal/privacy-policy"
+  },
+  "categories": ["developer-tools", "infrastructure"],
```
Comment (cristinabuenahora) [inline integrations/cortex/manifest.json:14]: @InTheCloudDan Thoughts on adding an "internal-developer-portal" category?

Comment (InTheCloudDan) [reply integrations/cortex/manifest.json:14]: @cristinabuenahora I'll chat with our Docs team about it. There is additional work we're going to be doing to align our tags in-app with better categorization but I don't want to block this PR on it. If we do add it I'll come back and update!

**Review by InTheCloudDan — APPROVED** [review]

**Pull request merged**